dev/NavigationView/NavigationView.h

```diff
@@ -75,22 +75,87 @@ class NavigationView :
 
     void CoerceToGreaterThanZero(double& value);
 
-    void OnNavigationViewItemInvoked(const winrt::NavigationViewItem& nvi);
+    void OnRepeaterElementPrepared(const winrt::ItemsRepeater& ir, const winrt::ItemsRepeaterElementPreparedEventArgs& args);
+    void OnRepeaterElementClearing(const winrt::ItemsRepeater& ir, const winrt::ItemsRepeaterElementClearingEventArgs& args);
+
+    com_ptr<NavigationViewItemsFactory> m_navigationViewItemsFactory{ nullptr };
```

Comment:
>m_navigationViewItemsFactory [](start = 40, length = 28)

This member variable has to be public?

Reply:
It is being used from NavigationViewItem. Please make the field private and expose a read only method.